Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -45,6 +45,9 @@ public enum BTPayPalError: Error, CustomNSError, LocalizedError, Equatable {
     /// 13. Missing PayPal Request
     case missingPayPalRequest
 
+    /// 14. Invalid authorization type
```

Comment:
small nit: take it or leave it
```suggestion
    /// 14. Invalid Authorization Type
```